go.mod

```
@@ -1,16 +1,60 @@
 module github.com/openshift/openshift-state-metrics
 
-go 1.14
+go 1.19
 
 require (
 	github.com/openshift/api v0.0.0-20200623075207-eb651a5bb0ad
 	github.com/openshift/client-go v0.0.0-20200623090625-83993cebb5ae
 	github.com/prometheus/client_golang v1.12.1
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/net v0.0.0-20210525063256-abc453219eb5
+	golang.org/x/net v0.0.0-20220722155237-a158d28d115b
```

```suggestion
	golang.org/x/net v0.4.0
```
Any reason to not pin this to `v0.4.0`?

---

[Replaced it earlier](https://github.com/openshift/openshift-state-metrics/pull/100/commits/c1e4aa319bacf27e581f69c5f6649f5f7b9e7277#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L60) but I've moved this to require now to allow updation to v0.4.0+ too. 👍🏼